src/components/AvatarGroup/SbAvatarGroup.vue

```diff
@@ -102,8 +102,11 @@ export default {
       }
     },
 
-    handleMoreAvatarsClick(event, vm) {
-      this.$emit('click', event, vm)
+    handleMoreAvatarsClick() {
+      this.isVisibleDropdown = !this.isVisibleDropdown
```

Comment:
`this.isVisibleDropdown` was never updated in this component before and was never used.
We keep it updated now as its being used in the event payload